### PR TITLE
feature/kotlin-1.4 -> master

### PR DESCRIPTION
**Pull request by romrell4**
This change is [Reviewable](https://reviewable.io/reviews/byu-oit/kotlin-uapi/184)

### CODE REVIEW

File: core/schemagen/src/main/kotlin/JacksonSchemaGenerator.kt [hunk 1]
```diff
@@ -18,11 +18,11 @@ private val defaultObjectMapper by lazy { ObjectMapper().apply {
 
 class JacksonSchemaGenerator(mapper: ObjectMapper = defaultObjectMapper): SchemaGenerator {
     private val generator by lazy { JsonSchemaGenerator(mapper) }
-    override fun generateSchemaFor(type: KClass<*>): Schema {
+    override fun generateSchemaFor(type: KClass<*>): Schema? {
         return generator.generateJsonSchema(type.java).toSchema()
     }
 
-    private fun JsonNode.toSchema(): Schema {
+    private fun JsonNode.toSchema(): Schema? {
```
Comment (jvisker) [inline core/schemagen/src/main/kotlin/JacksonSchemaGenerator.kt:25]: Why?

Comment (jvisker) [reply core/schemagen/src/main/kotlin/JacksonSchemaGenerator.kt:25]: This seems unrelated to kotlin 1.4

Comment (romrell4) [reply core/schemagen/src/main/kotlin/JacksonSchemaGenerator.kt:25]: I updated the entire parent pom. This was a change in Jackson from months ago

**Review by jvisker — APPROVED** [review]
I'm a little confused on how making that nullable didn't break anything, but if it works I guess I am okay with it.

> Reviewed 30 of 30 files at r1.
> *[Reviewable](https://reviewable.io/reviews/byu-oit/kotlin-uapi/184#-:-MGuDIUS-hm7DUUCpj1R:bxdwxpq)* status: :shipit: complete! all files reviewed, all discussions resolved

___
*[core/schemagen/src/main/kotlin/JacksonSchemaGenerator.kt, line 25 at r1](https://reviewable.io/reviews/byu-oit/kotlin-uapi/184#-MGuCnumH_40ozQUwAlL-r1-25:-MGuD1NtEsnCQ7oH95pI:bgb7yin) ([raw file](https://github.com/byu-oit/kotlin-uapi/blob/c7850a9d929c2164e9adc9b5b6553e6e2bb941c8/core/schemagen/src/main/kotlin/JacksonSchemaGenerator.kt#L25)):*
<details><summary><i>Previously, romrell4 (Eric Romrell) wrote…</i></summary><blockquote>

I updated the entire parent pom. This was a change in Jackson from months ago
</blockquote></details>

Gotcha. So unrelated to kotlin specifically.

**Pull request merged**